src/app/machines/views/MachineDetails/MachineNetwork/EditBondForm/EditBondForm.test.tsx

```diff
@@ -280,65 +283,51 @@ describe("EditBondForm", () => {
         interfaces,
       }),
     ];
-    const store = mockStore(state);
-    // Use a component to pass props to the form so that setProps can be used
-    // below.
-    const PassthroughComponent = ({ ...props }) => (
-      <Provider store={store}>
-        <MemoryRouter
-          initialEntries={[{ pathname: "/machines", key: "testKey" }]}
-        >
-          <CompatRouter>
-            <EditBondForm
-              close={jest.fn()}
-              nic={nic}
-              selected={[
-                { nicId: interfaces[0].id },
-                { nicId: interfaces[1].id },
-              ]}
-              setSelected={jest.fn()}
-              systemId="abc123"
-              {...props}
-            />
-          </CompatRouter>
-        </MemoryRouter>
-      </Provider>
+    const { rerender } = renderWithBrowserRouter(
+      <EditBondForm
+        close={jest.fn()}
+        nic={nic}
+        selected={[{ nicId: interfaces[0].id }, { nicId: interfaces[1].id }]}
+        setSelected={jest.fn()}
+        systemId="abc123"
+      />,
+      { route: "/machines", state }
     );
-    const wrapper = mount(<PassthroughComponent />);
-    wrapper.find("button[data-testid='edit-members']").simulate("click");
-    await waitForComponentToPaint(wrapper);
+    expect(
+      screen.getByRole("button", { name: "Save interface" })
+    ).not.toBeDisabled();
```

Comment:
You seem to have an additional assertion when comparing to the previous version of this test. 
```suggestion
```
Also, shouldn't the button be disabled at this point?

Reply:
Good point - I've updated the test to check that it *is* disabled.